package.json

```diff
@@ -39,8 +39,10 @@
     "script"
   ],
   "dependencies": {
+    "each-async": "^1.1.1",
     "meow": "^3.0.0",
     "osx-trash": "^1.0.0",
+    "path-exists": "^1.0.0",
```

Comment:
I prefer to duplicate deps. If I suddenly don't need it in the CLI and remove it, tests will fail. Like everything I would prefer for deps to be explicit.